src/App.vue

```diff
@@ -84,6 +84,7 @@
 
 .acrylic {
 	backdrop-filter: blur(5px);
+	background: rgba(var(--v-theme-primary), 0.75) !important;
```

Comment:
maybe 0.67

Reply:
![image](https://user-images.githubusercontent.com/6266356/182222566-ed16e33e-3a1f-4dc2-819a-f8b825a1ef71.png)

ok?

Reply:
![image](https://user-images.githubusercontent.com/6266356/182222634-b2f89adb-a88c-4f14-9fd4-d461bdc321e3.png)

0.5 might be too transparent

Reply:
Yeah, 0.66 looks good.